src/ui/units/wizard/actions/dialog.ts

```diff
@@ -122,10 +122,11 @@ export function openDialogPlaceholder({placeholder, onApply}: OpenDialogPlacehol
 
                             dispatch(closeDialog());
 
-                            dispatch(updatePreviewAndClientChartsConfig({}));
-
                             if (onApply) {
                                 onApply();
+                            } else {
+                                // Else call default onUpdate action
+                                dispatch(updatePreviewAndClientChartsConfig({}));
```

Comment:
It looks like a bug
You either need to remove the update altogether, or always apply it